### PR TITLE
new feature: we can disable lapi and specify remote one 

### LINKED ISSUE #20: crowdsec version needs to be updated to v1.4.1
Is there a way to update crowdsec ourselves or do we need to wait for updates?

v1.3.4 is currently installed but the latest version v1.4.1

Thanks

### PR DESCRIPTION
**Pull request by he2ss**
+ add new feature : disable lapi and specify remote one
+ update crowdsec version

(fix #20 and #19 )

### CODE REVIEW

**Review by sabban — APPROVED** [review]
lgtm

File: crowdsec/DOCS.md [hunk 1]
```diff
@@ -59,6 +63,22 @@ disable_online_api: false
 Acquisition config file for crowdsec ([see documentation](https://docs.crowdsec.net/docs/next/concepts/#acquisition)).
 The default acquisition allow Crowdsec add-on to process all logs from the host system journal.
 
+### Option: `disable_lapi` (optional)
+
+Disable local API (if you want only the agent).
+
+### Option: `remote_lapi_url` (optional)
+
+When `disable_lapi` setted to `true`, you need to specify the remote local API URL.
```
Comment (blotus) [inline crowdsec/DOCS.md:72]: `is set` instead of `setted`

File: crowdsec/DOCS.md [hunk 2]
```diff
@@ -59,6 +63,22 @@ disable_online_api: false
...
+
+### Option: `agent_username` (optional)
+
+When `disable_lapi` setted to `true`, you need to specify the agent username to connect to the remote local API.
```
Comment (blotus) [inline crowdsec/DOCS.md:76]: `is set` instead of `setted`

File: crowdsec/DOCS.md [hunk 3]
```diff
@@ -59,6 +63,22 @@ disable_online_api: false
...
+
+### Option: `agent_password` (optional)
+
+When `disable_lapi` setted to `true`, you need to specify the agent password to connect to the remote local API.
```
Comment (blotus) [inline crowdsec/DOCS.md:80]: `is set` instead of `setted`

**Pull request merged**